modern/src/resources/l10n/en.json

```diff
@@ -231,6 +231,7 @@
     "linkGoogleMaps": "Google Maps",
     "linkAppleMaps": "Apple Maps",
     "linkStreetView": "Street View",
+    "positionCoolantTemp": "Coolant Temperature",
```

Comment:
Can you please move this down to the list, ideally in the same order it is present in the Position.java.